corehq/apps/geospatial/templates/gps_capture.html

```diff
@@ -9,10 +9,6 @@
                 {% blocktrans %}
                     Mobile workers that do not have any associated GPS data.
                 {% endblocktrans %}
-            {% else %}
-                {% blocktrans %}
-                    Cases that do not have any associated GPS data.
-                {% endblocktrans %}
```

Comment:
These are now removed since the user would now load cases missing GPS data or All cases.